src/Service/ElementService.php

```diff
@@ -15,20 +17,46 @@
 
 namespace Pimcore\Bundle\AdminBundle\Service;
 
-use Pimcore\Bundle\AdminBundle\CustomView\Config;
+use Pimcore\Config;
+use Pimcore\Bundle\AdminBundle\CustomView;
+use Pimcore\Bundle\AdminBundle\Event\ElementAdminStyleEvent;
+use Pimcore\Logger;
+use Pimcore\Model\Asset;
+use Pimcore\Model\DataObject;
+use Pimcore\Model\Document;
+use Pimcore\Model\Element\ElementInterface;
+use Pimcore\Model\Element\Service;
+use Pimcore\Model\Site;
+use Pimcore\Security\User\UserLoader;
+use Pimcore\Tool\Frontend;
+use Pimcore\Bundle\AdminBundle\Controller\Traits\AdminStyleTrait;
+
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
-class ElementService
+/**
+ * @internal
+ */
```

Comment:
Not sure, can we just turn into `internal` without a warning period eg. `deprecation`?